test/integration/golden_response/staging/get_stat_ranking/california.json

```diff
@@ -332,12 +332,12 @@
           {
             "rank": 7,
             "value": 50.5,
-            "placeDcid": "geoId/06057"
+            "placeDcid": "geoId/06005"
```

Comment:
should we expect the ranking to change from code update? @pradh

Reply:
This is basically a swap of ranks when value is same (7<->8) due to some non-determinism in ranking that we discussed yesterday in https://github.com/datacommonsorg/mixer/pull/452